src/components/SiteInformation/SiteInformation.tsx

```diff
@@ -0,0 +1,21 @@
+import { Alert, TextLink } from '@/component-library';
+
+const SiteInformation = (): JSX.Element => {
+  const hasLink = !!process.env.REACT_APP_SITE_INFORMATION_LINK;
+
+  return (
+    <Alert status='warning'>
+      {process.env.REACT_APP_SITE_INFORMATION_MESSAGE}
+      {hasLink && (
+        <>
+          {' '}
```

Comment:
This nbsp didn't render inside the TextLink component, so I've had to use a fragment.